src/Chirp.DBService/Chirp.DBService.csproj

```diff
@@ -13,13 +13,10 @@
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       </PackageReference>
       <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="7.0.12" />
-    </ItemGroup>
-
-
-
+    </ItemGroup>    
+    
     <ItemGroup>
-      <ProjectReference Include="..\Chirp.Utilities\Chirp.Utilities.csproj" />
+        <ExcludeFromCodeCoverage Include="src\Chirp.DBService\Migrations"/>
```

Comment:
We also want to exclude our `Chirp.DBService\DbInitializer.cs`, as DB seeding is not something we need to test.